src/consensus/vote_tracker.zig

```diff
@@ -0,0 +1,343 @@
+const std = @import("std");
+const sig = @import("../sig.zig");
+const builtin = @import("builtin");
+
+const Slot = sig.core.Slot;
+const Hash = sig.core.Hash;
+const Pubkey = sig.core.Pubkey;
+
+/// TODO: these decls should be moved into their own actual namespaces in the codebase eventually.
+const commitment = struct {
+    pub const VOTE_THRESHOLD_SIZE: f64 = 2.0 / 3.0;
+};
+
+pub const VoteTracker = struct {
```

Comment:
Noticed the implementation does not include `get_slot_vote_tracker`. Is that intentional?

Reply:
I have it implemented locally on the top PR, I've pulled it down here now:
a69ed52cb91ace5bea5c7399a0476aac785cecfa

Reply:
Seems a comment was unintentionally committed

> // self.slot_vote_trackers.read().unwrap().get(&slot).cloned()